test/services/LibraryService.test.ts

```diff
@@ -0,0 +1,48 @@
+import { initialize, Server } from '@projecttacoma/node-fhir-server-core';
+import { serverConfig } from '../../src/config/serverConfig';
+import { cleanUpDb, testSetup } from '../utils';
+import supertest from 'supertest';
+
+let server: Server;
+
+const Library: fhir4.Library = {
+  resourceType: 'Library',
+  type: { coding: [{ code: 'logic-library' }] },
+  id: 'library123',
+  status: 'active'
+};
+
+describe('LibraryService', () => {
+  beforeAll(async () => {
+    const config = serverConfig;
+    server = initialize(config);
```

Comment:
Also, is this really necessary?

```suggestion
    server = initialize(serverConfig);
```